src/nvim_bridge.rs

```diff
@@ -282,6 +288,13 @@ pub enum RedrawEvent {
 
     Ignored(String),
     Unknown(String),
+
+    MessageShow(MessageEntry, bool),
+    MessageShowMode(Vec<(u64, String)>),
+    MessageShowCmd(Vec<(u64, String)>),
```

Comment:
@smolck This is the event that shows those command indicators we were seeing on https://github.com/vhakulinen/gnvim/issues/53